untangle/mfw_schema
# MFW-4523 Added a mteric field to route schema

Problem: Add 'metric' property in static routes schema

Changes: added a new field 'metric' to route schema

Testing instructions:
Run the test cases by using command ./validate.py in mfw_schema
Check all test cases passed or not

Expected Result: 
All the test cases from route_schema should pass
Please refer below output for reference
```
=== START OF MODULE TEST: <module 'v1.routes.validate_routes' from '/home/vishal/Desktop/bug/4523/mfw_schema/v1/routes/validate_routes.py'> ===

WARNING: Using default json filename:/home/vishal/Desktop/bug/4523/mfw_schema/v1/routes/test_routes.json
WARNING: Using default schema filename:/home/vishal/Desktop/bug/4523/mfw_schema/v1/routes/test_schema.json
test_routes (v1.routes.validate_routes.TestRoutesSchema)
validates routes ... ok

----------------------------------------------------------------------
Ran 1 test in 0.001s

OK
=== END OF MODULE TEST: <module 'v1.routes.validate_routes' from '/home/vishal/Desktop/bug/4523/mfw_schema/v1/routes/validate_routes.py'> ===

```
